opm/grid/common/GridPartitioning.cpp

```diff
@@ -402,11 +402,44 @@ void addOverlapLayer(const CpGrid& grid, int index, const CpGrid::Codim<0>::Enti
         }
     }
 
+    void addOverlapLayerNoZeroTrans(const CpGrid& grid, int index,
+                                    const int owner, const std::vector<int>& cell_part,
+                                    std::vector<std::tuple<int,int,char>>& exportList,
+                                    int recursion_deps, const double* trans)
+    {
+        using AttributeSet = Dune::OwnerOverlapCopyAttributeSet::AttributeSet;
+        for (int loc_face = 0; loc_face < grid.numCellFaces(index); ++loc_face) {
+            int face = grid.cellFace(index, loc_face);
+            int faceCell0 = grid.faceCell(face, 0);
+            int faceCell1 = grid.faceCell(face, 1);
```

Comment:
Are we sure that there always is a neighbor?

Reply:
If face is on the boundary otherCell == -1.